### PR TITLE
Balanced CIFAR10 Dataset

### PR DESCRIPTION
**Pull request by jmduarte**
- Adding balanced CIFAR10 dataset for final performance (code to get indices + index numpy array file)
- With this balanced dataset, the accuracy is higher (and more similar to the accuracy on the full dataset)

| Model | Datset | Accuracy |
| ---  | --- | --- |
| Pretrained ResNet  | Full test set | 87.2% | 
| Pretrained ResNet | Original subset | 85.5% |
| Pretrained ResNet | Balanced subset | 87.0% |
| Pretrained ResNet TFLite quantized |  Full test set | 87.0% |
| Pretrained ResNet TFLite quantized | Original subset | 86.5% |
| Pretrained ResNet TFLite quantized | Balanced subset | 87.0% |

@cskiraly @maltanar @jeremy-syn

### CODE REVIEW

**Comment by maltanar** [comment]
Using `PERF_SAMPLE=True` and `QUANT_MODEL=True` in `tflite_test.py` (which were the default values) I was able to reproduce the 87.0% accuracy with the new balanced subset.

Very minor/unrelated-to-PR comment: README specifies `# Prepare Python venv (Python 3.6+ and pip>20 required)` but `scipy==1.6.0` in `requirements.txt` in fact requires Python 3.7+ as specified here so I had to install that for the reproduction: 

https://scipy.github.io/devdocs/dev/toolchain.html#numpy

**Comment by maltanar** [comment]
I've also signed the CLA now.

**Comment by cskiraly** [comment]
@jmduarte can you also sign the CLA (or let me know if you have already signed)
Thanks

File: benchmark/training/image_classification/test.py [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
 import pickle
 import tensorflow as tf
-import keras
 from sklearn.metrics import roc_auc_score
 
 import train
 import eval_functions_eembc
 import keras_model
 
+PERF_SAMPLE = True
```
Comment (cskiraly) [inline benchmark/training/image_classification/test.py:21]: Can you add a comment on what True/False means? Are the scripts still working with PERF_SAMPLE = False?

Comment (PieMonty) [reply benchmark/training/image_classification/test.py:21]: It looks like it's a type we can remove, together with line 32.

Comment (jmduarte) [reply benchmark/training/image_classification/test.py:23]: This boolean switches between using the official MLPerf Tiny subset for validation (`True`) or the full CIFAR10 validation set (`False`). I've added a comment to that effect now.

**Comment by jmduarte** [comment]
> @jmduarte can you also sign the CLA (or let me know if you have already signed) Thanks

I have signed the CLA, I think the job is failing from @maltanar. Maybe he needs to link his GitHub to the CLA appropriately? 

```
Created a committerMap for users in PR: (signed: jmduarte-4932543, ) , (notSigned:  maltanar-1880902, )
```

**Review by cskiraly — APPROVED** [review]
LGTM

**Comment by cskiraly** [comment]
@jmduarte @maltanar , what do you think, should we also update the performance threshold in https://github.com/mlcommons/tiny/blob/master/benchmark/README.md
and in https://github.com/mlcommons/tiny/blob/master/benchmark/MLPerfTiny_Rules.adoc ?
